### ISSUE TITLE
Validator: invalid ERROR on Object.keys collection

### ISSUE THREAD

**Issue by rodionos** [kirmanak/Axibase-Charts-Syntax#179]
This is a valid syntax.
It's now reporting 'object is unknown'

```
var apps = {
"abc"	: {"Region": "ABC", "App": "app1"},
"cde"	: {"Region": "CDE", "App": "app2"}
}
endvar

  for agent in Object.keys(apps)

   [series]
    entity = @{agent}

  endfor
```

**Comment by kirmanak** [comment]
@rodionos What is allowed here?
1. `Object.keys({var name})`
2. `{var name}`
3. `{var name}[{index}]`
What else?

**Comment by rodionos** [comment]
Please check sources what is accepted by `in` operator. There should be a better way to check than relying simply on regex. It needs to be an iteratable javascript object first of all.

**Comment by kirmanak** [comment]
Fixed in #181
